### PR TITLE
fix(DI): class namespace and services loading

### PR DESCRIPTION
**Pull request by mremi**
I am targeting this branch, because this fixes broken DI.

## Changelog

```markdown
### Fixed
Fix class namespace and services loading
```

## Subject

This PR fixes broken stuff due to last migration from SonataCoreBundle.

### CODE REVIEW

File: src/Bridge/Symfony/Resources/config/services.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="sonata.doctrine.model.adapter.chain" class="Sonata\Doctrine\Model\Adapter\AdapterChain" public="true"/>
+        <service id="sonata.doctrine.model.adapter.chain" class="Sonata\Doctrine\Adapter\AdapterChain" public="true"/>
```
Comment (kunicmarko20) [inline src/Bridge/Symfony/Resources/config/services.xml]: I guess no one used this until now :sweat_smile:

**Comment by greg0ire** [comment]
Thanks @mremi !

**Comment by mremi** [comment]
You're welcome, could you please bump a new release?

**Comment by greg0ire** [comment]
Will do tonight

**Pull request merged**